### DISCUSSION TITLE
Workaround for tui input number prefix overlap

### DISCUSSION THREAD

**Discussion by Joshua-Ryan** [taiga-family/taiga-ui#3016] [Q&A] [answered]
Hi everyone,

Does anyone know of a work around for the input number component's prefix bug? 

The component works as expected some times, but when patching the value of the input number, it can occur in the prefix (in my case "$") to overlap the first number of the component's value. I have noticed that the bug was noted and fixed back in June but almost immediately regressed (prs below). 

Best regards,
Josh
-----------
original fix pr: https://github.com/taiga-family/taiga-ui/issues/1862

Regression pr: https://github.com/taiga-family/taiga-ui/issues/2019

picture of issue:
<img width="344" alt="Screen Shot 2022-11-03 at 4 05 13 PM" src="https://user-images.githubusercontent.com/48114022/199823204-5f5751fe-0dfa-4dea-897c-c9bb29785ef5.png">

**Comment by waterplea** [comment, marked as answer]
Please add comment with steps to reproduce to the original issue and we'll reopen it

**Reply by Joshua-Ryan** [reply]
Replication steps:
1. use ngif on a input number
2. conditionally show and then patch said component
3. Notice overlap of icon and first number

**Reply by waterplea** [reply]
Cannot seem to reproduce it on the latest code:
https://stackblitz.com/edit/angular-qymugb
